test_utils/factories.py

```diff
@@ -517,6 +520,26 @@ class Meta(object):
     cornerstone_base_url = factory.LazyAttribute(lambda x: FAKER.file_path())
 
 
+class CornerstoneGlobalConfigurationFactory(factory.django.DjangoModelFactory):
+    """
+    ``CornerstoneGlobalConfiguration`` factory.
+
+    Creates an instance of ``CornerstoneGlobalConfiguration`` with minimal boilerplate.
+    """
+
+    class Meta(object):
+        """
+        Meta for ``CornerstoneGlobalConfiguration``.
+        """
+
+        model = CornerstoneGlobalConfiguration
+
+    id = factory.LazyAttribute(lambda x: FAKER.random_int(min=1))
+    completion_status_api_path = factory.LazyAttribute(lambda x: FAKER.file_path())
+    oauth_api_path = factory.LazyAttribute(lambda x: FAKER.file_path())
+    subject_mapping = {}
```

Comment:
nit: we should have default value of `subject_mapping` in `CornerstoneGlobalConfigurationFactory`
may be this mapping
```
subject_mapping={
                "Computer Science": "Technology",
                "Communication": "Business Skills",
                "Music": "Creative",
                "Design": "Creative"
            }
```